Packs/ContentManagement/Scripts/CommitFiles/CommitFiles.py

```diff
@@ -236,8 +289,12 @@ def commit_git(git_integration: str, branch_name: str, content_file: ContentFile
         commit_content_item_gitlab(branch_name, content_file, new_files, modified_files)
     elif git_integration == 'GitHub':
         commit_content_item(branch_name, content_file, new_files, modified_files)
-    else:  # git_integration == 'Bitbucket'
+    elif git_integration == 'Bitbucket':
```

Comment:
consider changing it to a [switch case](https://learnpython.com/blog/python-match-case-statement/)